### PR TITLE
#134: update gradle wrapper version to 3.4.1

### PR DESCRIPTION
**Pull request by andrii-kovalenko-ct**
@ahalberkamp @heshamMassoud @lojzatran @LEQADA 

I updated the gradle wrapper to the most recent version cos the old one causes the problems in IDEA tests debug output, see #134 for more details.

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -30,6 +48,7 @@ die ( ) {
 cygwin=false
 msys=false
 darwin=false
+nonstop=false
```
Comment (heshamMassoud) [inline gradlew]: the `msys` flag is not needed I guess, no?

File: gradlew.bat [hunk 2]
```diff
@@ -60,11 +59,6 @@ set _SKIP=2
 if "x%~1" == "x" goto execute
 
 set CMD_LINE_ARGS=%*
-goto execute
-
-:4NT_args
```
Comment (heshamMassoud) [inline gradlew.bat]: why removed?

**Comment by andrii-kovalenko-ct** [comment]
all this changes are done automatically by executing the command:
```
gradle wrapper --gradle-version 3.4.1
```
so, basically all new incoming changes are default Gradle values for this version of the wrapper, e.g. that's how the wrapper should looks if you create a new project

File: gradlew (hunk 1 above)
Comment (andrii-kovalenko-ct) [inline gradlew]: https://github.com/commercetools/commercetools-payone-integration/pull/136#issuecomment-286792309

File: gradlew.bat (hunk 2 above)
Comment (andrii-kovalenko-ct) [inline gradlew.bat]: https://github.com/commercetools/commercetools-payone-integration/pull/136#issuecomment-286792309

Comment (heshamMassoud) [inline gradlew.bat]: @andrii-kovalenko-ct ah ok. Then, I guess there is not much to review :)

**Pull request merged**